charts/sscs-tribunals-api/values.yaml

```diff
@@ -77,10 +88,23 @@ java:
     ELINKS_V2_FEATURE_ENABLED: false
     DELETED_REDACTED_DOC_ENABLED: false
     UPLOAD_HEARING_FILTER_ENABLED: false
+    IDAM_OAUTH2_CLIENT_ID: "sscs"
+    CORE_CASE_DATA_JURISDICTION_ID: SSCS
+    CORE_CASE_DATA_CASE_TYPE_ID: Benefit
+    SEND_LETTER_SERVICE_BASEURL: "http://rpe-send-letter-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    SEND_LETTER_SERVICE_ENABLED: "true"
+    TRUST_ALL_CERTS: "true"
```

Comment:
just a general observation some env vars uses "" and some without "" may be we can make them consistent while you are here :)